ALLHUBS-Jan-2024-Liftoff/code-stars
# 20 save as pdf

In this branch, I:
- Created a save as PDF button
- Added it to the campground page
- Styled it with Bootstrap

To test it:
1. Run npm install in VSCode
2. Start both servers
3. Click the Save as PDF button on any campground page. It should create a pdf snapshot with the campground info and rating.
